mysql-test/suite/rocksdb.rpl/combinations

```diff
@@ -12,3 +12,25 @@ binlog-format=statement
 
 [mix]
 binlog-format=mixed
+
+[row-write-committed-native-partitioning]
+mtr-result-dir = r-native-partitioning
+mtr-tests-list = rpl_rocksdb_extra_col_slave
```

Comment:
Is it possible to have something like "mtr-tests-list = *" here or omit it altogether for the same effect?

Reply:
Yes, if that option is omitted, all tests from the suite will take part in the combination testing. There can be the situation when, for example, for one set of options we need to test the whole suite (like 'write-committed' and 'write-prepared' in MyRocks), but for the other set of options we need to test only the certain tests(which are selected in this option).